### ISSUE TITLE
Question about batch size in the caso of training in FMB dataset

### ISSUE THREAD

**Issue by iacopo97** [LiJiahang617/Road-Former#7]
Good morning, thank you for this work. I would like to know what is the exact batch size that you used for FMB dataset, from the config file it seems 4. If it is the case, how many gpus have you used?

**Comment by iacopo97** [comment]
I would like to know if the checkpoint that you considered is the one with best metric on validation set or the one obtained at the end of the training in the last epoch?

**Comment by jianxinhuang3** [comment]
All experiments were conducted using a single 3090 GPU, and the batch size was set to the maximum allowed by the GPU memory. As indicated in the config, for the FMB dataset with ConvNeXt-B as the backbone, the batch size was set to 2.
﻿
Additionally, the checkpoint used was the one with the best metric on the validation set, not the one obtained at the end of training.

**Comment by iacopo97** [comment]
Thank you very much for the kind answer
